tools/greenbone-nvt-sync.in

```diff
@@ -20,6 +20,16 @@
 # This script updates the local Network Vulnerability Tests (NVTs) from the
 # Greenbone Security Feed (GSF) or the Greenbone Community Feed (GCF). 
 
+if [ "`id -u`" -eq "0" ]
+then
+  echo "$0 must not be executed as privileged user root"
```

Comment:
I think this should go to stderr.

Maybe also relevant, almost all the other error exits also do a log_err (and use stderr_write instead of echo).

Reply:
I changed this accordingly.